src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/executor_pa.cpp

```diff
@@ -2371,8 +2371,17 @@ struct AttentionExecutor : public PagedAttentionExecutor {
                 auto SV = v_cache.m_dims[3];
                 auto Hk = k_cache.m_dims[1];  // shape: [block, H, 32, S]
                 parallel_for2d(Hk, zero_tokens, [&](size_t h, size_t l) {
-                    std::memset(k_cache.ptr_v(block_number, h, block_offset + l, 0), 0, S * k_cache.m_element_size);
-                    std::memset(v_cache.ptr_v(block_number, h, block_offset + l, 0), 0, SV * v_cache.m_element_size);
+                    auto set_zero =
+                        [](PlainTensor& cache, size_t block_number, size_t h, size_t l, size_t hidden_dims) {
+                            auto sub_byte_multiplier = get_sub_byte_multiplier(cache.get_precision());
+                            size_t cache_stride =
+                                (block_number * cache.stride(0) + h * cache.stride(1) + l * cache.stride(2)) *
+                                cache.get_precision().size() / sub_byte_multiplier;
+                            auto cahce_ptr = cache.m_ptr.get() + cache_stride;
```

Comment:
Typo: `cahce_ptr -> cache_ptr`.
There are several places for computing ptr/strides of u4 type plain_tensor which sharing same logic. Maybe we'd better to add u4 etc. support for it to refactor this in the future.